spec/lib/zipkin_sender_base_spec.rb

```diff
@@ -76,17 +76,16 @@
       tracer.end_span(span)
       expect(span.to_h).to eq(span_hash.merge(duration: 1_000_000))
     end
-    it 'flush if kind is SERVER in this span' do
-      span.kind = Trace::Span::Kind::SERVER
-      expect(tracer).to receive(:flush!)
-      expect(tracer).to receive(:reset)
-      tracer.end_span(span)
-    end
-    it 'flush if kind is CLIENT in this span and the span does not have parent span' do
-      span.kind = Trace::Span::Kind::CLIENT
-      expect(tracer).to receive(:flush!)
-      expect(tracer).to receive(:reset)
-      tracer.end_span(span)
+    [
+      Trace::Span::Kind::SERVER, Trace::Span::Kind::CLIENT,
+      Trace::Span::Kind::PRODUCER, Trace::Span::Kind::CONSUMER
+    ].each do |kind|
```

Comment:
any reason not to go with rspec's [shared examples](https://relishapp.com/rspec/rspec-core/v/3-9/docs/example-groups/shared-examples)?

Reply:
updated in 8859e64